aws/sqs_event_manager/sqs_event_manager/app.py

```diff
@@ -177,3 +177,78 @@ def process_message(record: Dict) -> Dict[str, Union[str, bool]]:
         result['status'] = f'{type(error).__name__}: {error}'
         logger.error(result['status'])
     return result
+
+
+def subscription_success(message: AWSSNSMessage) -> Dict:
+    """
+    Notify SCC on subscription successfully processed
+    """
+    request_data = basic_request(message)
+    request_data['offerIdentifier'] = message.offer_id
+    request_data['isFreeTrialTermPresent'] = message.free_trial_term_present
+    return request_data
+
+
+def subscription_removed(message: AWSSNSMessage) -> Dict:
+    """
+    Notify SCC on subscription successfully removed
+    """
+    request_data = basic_request(message)
+    request_data['offerIdentifier'] = message.offer_id
+    request_data['isFreeTrialTermPresent'] = message.free_trial_term_present
+    return request_data
+
+
+def subscription_failed(message: AWSSNSMessage) -> Dict:
+    """
+    Notify SCC on subscription process failed
+    """
+    request_data = basic_request(message)
+    request_data['offerIdentifier'] = message.offer_id
+    request_data['isFreeTrialTermPresent'] = message.free_trial_term_present
+    return request_data
+
+
+def entitlement_updated(message: AWSSNSMessage) -> Dict:
+    """
+    Notify SCC of customer entitlement change for the given product
+    """
+    return basic_request(message)
+
+
+def send_to_scc(
```

Comment:
If the service url is configurable maybe we want a generic name here and not tied to scc?

Reply:
we want it tied to SCC because in the last meeting we decided that any request from sqs manager has to go to SCC because SCC has to know about it. That's reason why I tied it to SCC. This will be the case for any SaaS offering now and in the future. We are not implementing special sauce without notifying SCC

Reply:
of course if you set URLs that are not pointing to SCC you can abuse the concept

Reply:
Since it's open source someone might want to use the code against something other than scc?

Reply:
ok, done